src/content/docs/logs/logs-everywhere/telemetry-in-context.mdx

```diff
@@ -0,0 +1,219 @@
+  FROM NrMTDConsumption SELECT latest(estimatedCost) WHERE productLine = 'DataPlatform' SINCE this month
+  ```
+
+  </Collapser>
+
+  <Collapser
+    id="nrql-alert"
+    title="NRQL alert: usage threshold example"
+  >
+
+  To create an alert when your usage exceeds a fixed monthly threshold for gigabytes of data, add this [NRQL query](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/) to your alert condition:
+
+  ```
+  FROM NrMTDConsumption SELECT latest(GigabytesIngested) WHERE productLine = 'DataPlatform'
+  ```
+
+  </Collapser>
+
+</CollapserGroup>
+
+## Ensure data privacy [#data-privacy]
+
+Our log management service automatically masks number patterns that appear to be for items such as credit cards or Social Security numbers. For more information, see our [security documentation](/docs/logs/get-started/new-relics-log-management-security-privacy) for log management.
+
+You can also use our obfuscation options to hash or mask sensitive data in your logs. This is critical, for example:
+
+* If it is impractical or impossible to split sensitive data into separate accounts
+* If you want data to be searchable and coherent without sensitive information exposed
+* If you want to restrict query capabilities for sensitive data for users without those access rights
+
+For more information, see our [obfuscation documentation](https://docswebsitedevelop-jsubiratdocswebsitedoc7721obfa.gtsb.io/docs/logs/ui-data/obfuscation-ui/). **REVIEWER: The obfuscation documentation is in a separate PR but you can see it from this link.** You can create, read, update, and delete obfuscation expressions and rules via the New Relic One UI or NerdGraph, our New Relic GraphQL explorer.
+
+## What's next? [#what-next]
+
+**REVIEWER: These bullets came from several different project docs. Not sure if they are all saying the same thing, just in different ways. :-) Are all these bullets needed to get customers excited to take action? Is this section even needed?**
+
+If you're troubleshooting an error with your app, you can easily see related network activity, Kubernetes pod health, cloud provider information, database storage, load balancer resources, logs, transaction data, spans and traces, and more, all in the context of the APM error.
+
+With telemetry in context, now your logs are:
+
+* **Time-sensitive.** See the counts and percentages of logs associated with your apps, hosts, and other related entities for a specific timeframe. Quickly gain context at that point in time.
+* **Precise.** Cut through the noise of thousands of logs when troubleshooting time-critical issues, so you automatically see only the most relevant logs. Navigate within multiple types of telemetry data and have them correlate back to the original issue.
+* **Security minded.** Understand context for security information and event management (SIEM) data across your infrastructure, network performance management, cloud platforms, and more.
```

Comment:
I'm not sure if this is quite relevant?

Reply:
Agreed. I was throwing a lot of separate pieces of info at this draft. :-)